src/EdozoLogo2/EdozoLogo2.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { EdozoLogo2 } from './EdozoLogo2';
+
+export default {
+  title: 'Components/EdozoLogo2',
+  component: EdozoLogo2,
+} as Meta;
+
+const Template: Story = args => <EdozoLogo2 {...args} />;
+
+export const Default = Template.bind({});
+
+export const WithName = Template.bind({});
+WithName.args = {
+  appName: 'Occupiers',
+};
+
+export const WithColourAndSmall = Template.bind({});
```

Comment:
Can you make sure there is a story for each of the colors. I can see you've got insight there and the occupier one is using maps color but just one hitting occupiers color i think